incubator/gitea/templates/configmap.yaml

```diff
@@ -0,0 +1,128 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  labels:
+    app: {{ template "fullname" . }}-config
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    component: {{ default "gitea" .Values.service.nameOverride }}
```

Comment:
I don't see why a `component` label on the configmap would make sense. I'd suggest you remove it.